### PR TITLE
VC-4218 processing set extra

### PR DESCRIPTION
**Pull request by alex-bodnar**
## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: _tests/database/accounts/extra_test.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+					Creator: "vcg1ljs7p2p9ae3en8knr3d3ke8srsfcj2zjvefv1g",
+					Hash:    "a935ea2c467d7f666ea2a67870564f2efb902c05f0a2bb4b6202832aedd26cd2",
+					Extras: []*extratypes.Extra{
+						{
+							Kind: extratypes.EXTRA_KIND_WALLET,
+							Data: "walletHolderName3",
+						},
+						{
+							Kind: extratypes.EXTRA_KIND_COMMENT,
+							Data: "coment 123",
```
Comment (DirusK) [inline _tests/database/accounts/extra_test.go:120]: Используй gofakeit
Там есть функции для генерации рандомных фраз и тд

File: _tests/database/accounts/extra_test.go [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+				accfilter: filter.NewFilter().SetArgument(accounts.FieldHash, "a935ea2c467d7f666ea2a67870564f2efb902c05f0a2bb4b6202832aedd26cd2"),
+			},
+			want: []*accountstypes.MsgSetExtra{
+				{
+					Creator: "vcg1ljs7p2p9ae3en8knr3d3ke8srsfcj2zjvefv1g",
+					Hash:    "a935ea2c467d7f666ea2a67870564f2efb902c05f0a2bb4b6202832aedd26cd2",
+					Extras: []*extratypes.Extra{
+						{
+							Kind: extratypes.EXTRA_KIND_WALLET,
+							Data: "walletHolderName3",
```
Comment (DirusK) [inline _tests/database/accounts/extra_test.go:116]: gofakeit  
Биткоин адресс или шо-то тип того можно использовать

File: _tests/database/accounts/extra_test.go (hunk 1 above)
Comment (alex-bodnar) [reply _tests/database/accounts/extra_test.go:120]: Нельзя, потому что я в следующем тесте достаю эти значения

File: _tests/database/accounts/extra_test.go (hunk 2 above)
Comment (alex-bodnar) [reply _tests/database/accounts/extra_test.go:116]: Нельзя, потому что я в следующем тесте достаю эти значения

**Comment by DirusK** [comment]
LGTM

**Review by vasylNaumenko — APPROVED** [review]
LGTM

**Pull request merged**